sfputil/main.py

```diff
@@ -415,7 +415,10 @@ def presence(port):
             if presence:
                 output_table.append([port_name, "Present"])
             else:
-                output_table.append([port_name, "Not present"])
+                if not platform_sfputil._is_valid_port(physical_port):
+                    output_table.append([port_name, "N/A"])
+                else:
+                    output_table.append([port_name, "Not present"])
```

Comment:
Maybe it makes more sense written as follows?

```python
            if platform_sfputil._is_valid_port(physical_port):
                status_string = "Present" if presence else "Not present"
                output_table.append([port_name, status_string])
            else:
                output_table.append([port_name, "N/A"])
```

Reply:
Hi, jleveque,

Your comment is applied.